indico/modules/events/client/js/display.js

```diff
@@ -26,10 +31,21 @@ import './favorite';
       });
     }
 
-    $(document).on('click', '[data-note-editor]', function(evt) {
-      evt.preventDefault();
-      openAjaxDialog($(this));
-    });
+    const containers = document.querySelectorAll('.edit-minutes-container');
+    if (containers) {
```

Comment:
Not sure if this `if` is needed: Assuming `containers` is array-like (iirc QSA returns some kind of DOM NodeList), it will never be falsy even if empty (it's not python 😟) - and `forEach` on an empty list simply doesn't do anything